docs/api-testing/environments.md

```diff
@@ -7,51 +7,63 @@ description: "Gain a wide range of options to mix and match your test settings w
+
+If properly set up, any API Testing test can be run against any environment. This can be achieved by turning the URL and any other part into variables that can be overwritten using the environments.  This allows you, for example, to set the default location as input and override those amounts with the environments feature without actually changing the Test.
+
+The **Environments** section lets you change anything, not just environments. You can run the Test against a certain environment, using a different API key and data source.
 
 
 ## What You'll Need
 * An existing API Testing Project and Test. For details on how to create them, see [API Testing Quickstart](/api-testing/quickstart/).
 * Recommended: review [Creating Reusable Variables and Snippets with the Vault](/api-testing/vault/)
 
-## Using Environments
 
-Once a domain is parametrized, you can override a variable, if needed.
+## Creating Environments
 
-To access and create new environments:
+Once a domain is parametrized, you can override a variable if needed.
 
-1. Visit the **Environment** section in the test project section:
-  <img src={useBaseUrl('img/api-fortress/2021/04/accessEnvironment.png')} alt="access environment"/>
+There are multiple ways to create and access Test environments:
 
-2. Add the details (typically the target domain) and select **Confirm**:
-  <img src={useBaseUrl('img/api-fortress/2021/04/createEnvironment.png')} alt="create environment"/>
+### From a Project
+1. From within a Project, go to the **Environments** section.<br/><img src={useBaseUrl('img/api-fortress/2021/04/accessEnvironment.png')} alt="access environment"/>
+2. Click **Create Environment**.
+3. Enter a name for your environment, then click **Save** when finished.<img src={useBaseUrl('img/api-fortress/2021/04/createEnvironment.png')} alt="create environment" width="300"/>
+4. From here, you can:
+   * Add variable(s) manually by clicking **New Entry** > Enter **Key** and **Value** pairs > **Confirm**.<br/><img src={useBaseUrl('img/api-fortress/2021/04/manualEntry.png')} alt="create environment" width="500"/>
+   * Import an existing environment by clicking **Import** > **Choose File** (accepted file types are Postman, .json, .csv).<br/><img src={useBaseUrl('img/api-fortress/2021/04/importEnv.png')} alt="create environment" width="500"/>
 
-3. Add the variable details and select **Confirm**:
-  <img src={useBaseUrl('img/api-fortress/2021/04/addDomainVar.png')} alt="variable in environment"/>
+ The resulting environment (along with its defined variables) will now appear each of your Tests as an option in the environment dropdown.
 
-  You can add multiple variables to each environment:
-  <img src={useBaseUrl('img/api-fortress/2021/04/result.png')} alt="result in environment"/>
+### From a Test
+1. Locate your desired Test from any of the following sections: **Tests**, **Compose**, **Schedule**, or **HTTP Client**. These are all of the places in the UI where you can run a Test.
```

Comment:
@kimsauce **Schedule** is not a section where you can run test(s) and it does not have the dropdown for the Environments. Even though the override section serves as a substitute for the Environments it doesn't show you the list of available environments so you have to enter manually the variables that could be new variables not strickly related to an environment you have in the dropdown. 
**HTTP Client** is not the place where you can run a test, you can execute a call there and then generate a test